classes/asserter/resolver.php

```diff
@@ -50,13 +52,28 @@ public function resolve($asserter)
 		{
 			$class = $this->checkClass($asserter);
 		}
-		else foreach ($this->namespaces as $namespace)
+		else
 		{
-			$class = $this->checkClass($namespace . '\\' . $asserter);
+			if (false === test::isValidIdentifier($asserter))
+			{
+				return null;
```

Comment:
We can but it's not valid ;)

What we can do is ask for `\\SomeAsserter` and this is what we try to check in the first `if`